system.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// StopEtcd stops etcd on a specific host
+func StopEtcd(node TestbedNode) error {
```

Comment:
That is fine. I just wanted to understand the usecase. I think we can keep these if you are using them in volplugin. It is good to have utilities in one place.

The way I am thinking these services will be started or stopped would be through systemd units. I am working on one for etcd right now for instance. Once we have that infrastructure in place, I think then we shall be able to call `systemctl start etcd`, `systemctl stop etcd` and so on in these utilitiy functions whichwill take care of setting up arguments and logging in consistent way.